cmd/config/get.go

```diff
@@ -0,0 +1,111 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pingidentity/pingctl/internal/logger"
+	"github.com/pingidentity/pingctl/internal/output"
+	"github.com/pingidentity/pingctl/internal/viperconfig"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v3"
+)
+
+func init() {
+	l := logger.Get()
+
+	l.Debug().Msgf("Initializing Config Get Subcommand...")
+}
+
+func NewConfigGetCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Get pingctl configuration settings.",
+		Long: `Get pingctl configuration settings.
+
+Example command usage: 'pingctl config get pingctl.color'`,
+		RunE: ConfigGetRunE,
+	}
+
+	return cmd
+}
+
+func ConfigGetRunE(cmd *cobra.Command, args []string) error {
+	l := logger.Get()
+	l.Debug().Msgf("Config Get Subcommand Called.")
+
+	viperKey, err := parseGetArgs(args, cmd)
+	if err != nil {
+		return err
+	}
+
+	// If the viper key is empty,
+	// the parseGetArgs() function already printed the entire configuration
+	if viperKey == "" {
+		return nil
+	}
+
+	// The only valid configuration keys are those that are already set in the
+	// configuration file. If the key is not recognized, return an error.
+	if !viper.InConfig(viperKey) {
+		validKeys := strings.Join(viperconfig.GetViperConfigKeys(), ", ")
```

Comment:
`viperconfig.GetViperConfigKeys()` here does not include valid object keys such as `pingone.worker`